### DISCUSSION TITLE
Suggestion - choice of openai models

### DISCUSSION THREAD

**Discussion by fuegovic** [danny-avila/LibreChat#84] [Feature Requests & Suggestions]
It would be really nice to be able to choose between different models like gpt-3.5-tubo, text-davinci-003, (the upcoming) gpt-4....

It could be as an option in the customGPT

Thanks, the project is amazing!

**Comment by danny-avila** [comment]
Thank you, glad you're enjoying it! This will be simple enough to get done this weekend

**Comment by fuegovic** [comment]
🎂
Thanks @danny-avila 
One year since I first posted here, I could not have imagined the journey I was embarking on! 😁
